package.json

```diff
@@ -11,7 +11,7 @@
     "@sentry/tracing": "^7.0.0",
     "@sentry/webpack-plugin": "^1.18.3",
     "@types/dompurify": "^2.0.3",
-    "@types/node": "^12",
```

Comment:
Only this isn't enough, and there's also this https://github.com/getsentry/develop/pull/60
Not sure if and where else I should change to fix this, and where I have to look into to have them match.